src/nyc_trees/apps/quiz/templates/quiz/quiz_complete_page.html

```diff
@@ -0,0 +1,39 @@
+{% extends "base_two_frames.html" %}
+
+{% load utils %}
+
+{% block aside %}
+    <div class="pageheading-description block">
+        <div class="row">
+            <div class="col-xs-12">
+                <a href="{% url 'training' %}" class="breadcrumb">
+                    <i class="icon-left-open-mini"></i>
+                    Training
+                </a>
+            </div>
+            <div class="col-xs-12">
+                <h2>{{ quiz.title }}</h2>
+            </div>
+        </div>
+    </div>
+{% endblock aside %}
+
+{% block main %}
+
+{% if passed_quiz %}
+<h3>Great job!</h3>
+<p>You answered {{ score }}/{{ quiz.questions|length }}
+    question{{ quiz.questions|length|pluralize }} correctly.</p>
+{# TODO: Where should this button go after completing the quiz? #}
+<p><a href="{% url 'training' %}" class="btn btn-primary">Finish</a></p>
+
+{% else %}
+<h3>Better luck next time...</h3>
```

Comment:
I'd ask @jfrankl and/or @dboyer for what text to use if the user fails the quiz.